lib/ex_aws/cloudfront/custom_policy.ex

```diff
@@ -0,0 +1,135 @@
+defmodule ExAws.CloudFront.CustomPolicy do
+  defstruct [:url, :date_less_than, :date_greater_than, :ip_address]
+
+  @doc """
+  Create a Custom Policy.
+  """
+  def create(url), do: create url, ExAws.Utils.now_in_seconds + 1800
+  def create(url, %DateTime{} = date_less_than), do: create url, date_less_than |> DateTime.to_unix
```

Comment:
Use parenthesis for the function call and consider avoiding `|>` for single pipes. It's hard to read how `|>` binds now.